_pages/2300_DocumentationGuidelines.md

```diff
@@ -20,7 +20,7 @@ Following the MSDN online help style and word choice helps developers find their
 **Tip:** The tool [GhostDoc](http://submain.com/products/ghostdoc.aspx) can generate a starting point for documenting code with a shortcut key.
 
 ### <a name="av2310"></a> Avoid inline comments  (AV2310) ![](/assets/images/2.png)
-If you feel the need to explain a block of code using a comment, consider replacing that block with a method with a clear name.
+If you feel the need to explain a block of code using a comment, consider replacing that block with a method or local function with a clear name.
```

Comment:
I'm not sure about this one. If the code becomes that much unreadable, a local function will not really help focusing on the current body. In that case, I would prefer to have a separate method or to properly encapsulate a concept that is now represented by primitive types. See also https://www.continuousimprover.com/2015/10/9-simple-practices-for-writing-better.html

Reply:
Makes sense. Let's discard this PR then.